samcli/lib/build/workflow_config.py

```diff
@@ -111,13 +111,23 @@ def get_workflow_config(runtime, code_dir, project_dir):
             JAVA_KOTLIN_GRADLE_CONFIG._replace(executable_search_paths=[code_dir, project_dir]),
             JAVA_MAVEN_CONFIG
         ]),
+
+         "provided": ManifestWorkflowSelector([
+            PYTHON_PIP_CONFIG,
+            RUBY_BUNDLER_CONFIG,
+            DOTNET_CLIPACKAGE_CONFIG,
+            NODEJS_NPM_CONFIG,
+        ]),
```

Comment:
Is there any reason why you did not add other builders?

Reply:
No particular reason, mainly just apprehension for testing.